cfgov/jinja2/v1/_includes/molecules/expandable.html

```diff
@@ -40,12 +40,11 @@
     %}
 {% endif %}
 {% set value = value or global_dict.expandable %}
-
 <div class="m-expandable
-            m-expandable__expanded
```

Comment:
This is intentional, as with this change the expandable state where it is expanded on page load will not work. See fourth expandable on `/browse-basic/`, which should be expanded on page load -->

![screen shot 2016-01-04 at 9 56 26 am](https://cloud.githubusercontent.com/assets/704760/12092191/8d1ff1c6-b2c9-11e5-928c-86f969d34b5f.png)